framework/source/class/qx/ui/window/IWindowManager.js

```diff
@@ -32,7 +32,9 @@ qx.Interface.define("qx.ui.window.IWindowManager",
      * @param desktop {IDesktop} The connected desktop
      */
     setDesktop : function(desktop) {
-      this.assertInterface(desktop, qx.ui.window.IDesktop);
+      if (desktop) {
```

Comment:
I think the parameter desktop should only be null or having the interface IDesktop implemented, so it should fail as well for undefined.

I'll change the PR to
```javascript
    if (desktop !== null) {
```